lists/global.csv

```diff
@@ -1115,5 +1115,7 @@ http://zoomshare.com/,HOST,Hosting and Blogging Platforms,2014-04-15,citizenlab,
 https://www.opentech.fund/,HUMR,Human Rights Issues,2017-09-26,OONI,
 https://www.womenonwaves.org/,HUMR,Human Rights Issues,2017-10-26,OONI,
 https://www.womenonweb.org/,HUMR,Human Rights Issues,2017-10-26,OONI,
+https://www.organicbouquet.com/,CULTR,Culture,2018-02-14,OONI,Valentine's Day site (some countries ban Valentine's Day)
```

Comment:
This site is ranked by alexa as number 1,946,858. Are we sure this is worthwhile to add to the global test list?

The site also only ships to the US, so maybe we don't want to have it in the global list.